lib/oli_web/pow/pow_helpers.ex

```diff
@@ -229,4 +201,45 @@ defmodule OliWeb.Pow.PowHelpers do
     |> Naming.humanize()
     |> String.downcase()
   end
+
+  defp providers_config_list() do
+    []
+    |> maybe_add_provider(:github)
+    |> maybe_add_provider(:google)
+  end
+
+  defp maybe_add_provider(providers_list, provider) do
+    client_id = Application.fetch_env!(:oli, :auth_providers)[:"#{provider}_client_id"]
+    client_secret = Application.fetch_env!(:oli, :auth_providers)[:"#{provider}_client_secret"]
+
+    if client_id != "" and client_secret != "" do
```

Comment:
This could also be nil right? (if they keys are not configured)

Reply:
Actually with the current code and config it shouldn't happen, but might cause some confusion in the future. Updated! Thanks 🙌